common/lib/capa/capa/util.py

```diff
@@ -173,8 +173,8 @@ def sanitize_html(html_code):
     attributes = bleach.ALLOWED_ATTRIBUTES.copy()
     # Yuck! but bleach does not offer the option of passing in allowed_protocols,
```

Comment:
As of bleach 1.5, this is no longer true: https://bleach.readthedocs.io/en/latest/clean.html#allowed-protocols-protocols .  Could you switch the code to pass the new `protocols` argument instead of mutating the default list?